src/test/java/cn/edu/tsinghua/tsfile/file/metadata/TimeSeriesMetadataTest.java

```diff
@@ -48,7 +48,7 @@ public void testConvertToThrift() {
   @Test
   public void testConvertToTSF() {
     for (DataType dataType : DataType.values()) {
-      TimeSeries timeSeries = new TimeSeries(measurementUID, dataType, deltaObjectType);
+      TimeSeries timeSeries = new TimeSeries(measurementUID, dataType, "");
```

Comment:
The definition of TimeSeries.thrift could be changed together in this pull request where the third parameter of TimeSeries is removed

Reply:
Will do it in another PR.